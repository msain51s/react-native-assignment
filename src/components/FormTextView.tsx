import React from 'react-native'
import { View,Text } from 'react-native'
import styles from '../utility/Styles'

type FormProps = {
    title:string,
    value:string
}

function FormTextView(props:FormProps){
    return(
        <View style={styles.formViewStyle}>
         <Text style={styles.textStyleMedium}>{props.title}</Text>
         <Text style={styles.textStyleMedium}>{props.value}</Text>
        </View>
    )
}

export default FormTextView