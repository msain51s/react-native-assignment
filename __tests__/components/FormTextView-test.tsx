import React, { useState } from "react";
import renderer from 'react-test-renderer'
import FormTextView from "../../src/components/FormTextView";

test('FormTextView render correctly',()=>{
    let setText:any
    const tree = renderer.create(<FormTextView title="key" value="value"/>)
    expect(tree).toMatchSnapshot()
})