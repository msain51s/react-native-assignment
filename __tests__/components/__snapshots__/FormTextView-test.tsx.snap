// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormTextView render correctly 1`] = `
<View
  style={
    {
      "flexDirection": "row",
      "justifyContent": "flex-start",
    }
  }
>
  <Text
    style={
      {
        "color": "black",
        "fontSize": 16,
        "margin": 10,
      }
    }
  >
    key
  </Text>
  <Text
    style={
      {
        "color": "black",
        "fontSize": 16,
        "margin": 10,
      }
    }
  >
    value
  </Text>
</View>
`;
